packages/auth/src/signature/middleware.test.ts

```diff
@@ -401,6 +401,101 @@ describe('Signature Service', (): void => {
       scope.isDone()
     })
 
+    test('middleware succeeds if BYPASS_SIGNATURE_VALIDATION is true with bad signature', async (): Promise<void> => {
+      const defaultByPassSignatureValidation = Config.bypassSignatureValidation
+      Config.bypassSignatureValidation = true
```

Comment:
This appears to work
```suggestion
      const config = await appContainer.container.use('config')
      config.bypassSignatureValidation = true
```